### PR TITLE
Upgrade to Web3 v1.2.0

### PR DESCRIPTION
**Pull request by BrendanChou**
Also still includes web3 1.0 because of `exchange-wrappers`

### CODE REVIEW

File: src/lib/BytesHelper.ts [hunk 1]
```diff
@@ -25,8 +25,8 @@ export function argToBytes(
     v = val.toFixed();
   }
 
-  return hexToBytes(
-    padLeft(toHex(v), 64, '0'),
+  return Web3.utils.hexToBytes(
+    Web3.utils.padLeft(Web3.utils.toHex(v), 64, '0'),
```
Comment (BrendanChou) [inline src/lib/BytesHelper.ts]: Actually may have to do the same thing I did above (add `.map(x => [x])`) to return `number[][]`
Not sure why this didn't cause an error.

Annoying how the contract types take in a `number[][]` for bytes arrays. It makes no sense.

File: src/modules/LimitOrders.ts [hunk 2]
```diff
@@ -325,23 +324,23 @@ export class LimitOrders {
   public getOrderHash(
     order: LimitOrder,
   ): string {
-    const structHash = soliditySha3(
+    const structHash = this.web3.utils.soliditySha3(
```
Comment (AntonioJuliano) [inline src/modules/LimitOrders.ts]: why are you using both `Web3.utils` and `this.web3.utils` in different places? We should probably just use `Web3.util` everywhere

File: package.json [hunk 3]
```diff
@@ -70,8 +70,7 @@
     "es6-promisify": "^6.0.1",
     "ethers": "^4.0.33",
     "openzeppelin-solidity": "2.1.3",
-    "web3": "1.0.0-beta.37",
-    "web3-utils": "1.0.0-beta.37"
+    "web3": "1.2.0"
```
Comment (AntonioJuliano) [inline package.json]: I can't comment on it, but did we also need to upgrade `typechain`?

File: package.json [hunk 4]
```diff
@@ -70,8 +70,7 @@
     "es6-promisify": "^6.0.1",
```
Comment (AntonioJuliano) [inline package.json]: publish a new version of solo itself too

**Pull request closed without merging**